search-app/test/cmr/search/test/unit/services/acls/collections_cache.clj

```diff
@@ -70,11 +94,18 @@
        test-coll1 (create-collection-for-gran-acls-test-entry "TEST_PROV2" "EntryTitle2" "C123-TEST_PROV2")]
   ;; populate the cache
   (hash-cache/set-value coll-by-provider-id-entry-title-cache coll-by-provider-id-entry-title-cache-key "C123-TEST_PROV2EntryTitle2" test-coll1)
+  (hash-cache/set-value coll-by-provider-id-entry-title-cache coll-by-provider-id-entry-title-cache-key "C456-TEST_PROV2EntryTitle2" {})
+
   (are3 [expected provider-id entry-title]
         (is (= expected (get-collection-gran-acls context provider-id entry-title)))
 
-        ;; "Collection not found in cache" -- TODO will add after update on next PR
-        ;; "Collection found in cache, but it is empty" -- TODO will add after update on next PR
+        "Collection not found in cache"
+        nil ;; expected
+        "C000-NON_EXISTENT" "EntryTitle"
```

Comment:
This test needs to change because I asked above that we look up the collection if it doesn't exist in the provider entry-title cache and return it.